### DISCUSSION TITLE
Storage Network traffic segregation

### DISCUSSION THREAD

**Discussion by miathedev** [longhorn/longhorn#6476] [Q&A]
Dear Longhorn Community,

Can Longhorn be configured to utilize a separate network dedicated to Longhorn-related traffic? 
The aim is to isolate this traffic from regular service traffic. 
I have already allocated an additional NIC, specifically "eth1," for this purpose.

I found "defaultSettings.storageNetwork" inside the Longhorn helm chart but couldn't see any documentation related how to use it.

With love,
Mia

**Comment by innobead** [comment]
You can refer to https://longhorn.io/docs/1.5.1/advanced-resources/deploy/storage-network/. 

cc @c3y1huang
